cmake/modules/AddSwift.cmake

```diff
@@ -638,6 +638,7 @@ endmacro()
 
 # Declare that files in this library are built with LLVM's support
 # libraries available.
-macro(set_swift_llvm_is_available)
-  add_compile_options(-DSWIFT_LLVM_SUPPORT_IS_AVAILABLE)
+macro(set_swift_llvm_is_available name)
```

Comment:
This should be changed to `function` from `macro` because it no longer pollutes the global namespace.

Reply:
agreed.